### PR TITLE
LRQA-75805 | master

### PR DESCRIPTION
**Pull request by weller0246**
https://issues.liferay.com/browse/LRQA-75805

### CODE REVIEW

**Comment by gaborlovasdotcom** [comment]
@weller0246 please SF

**Comment by weller0246** [comment]
ci:stop

**Comment by weller0246** [comment]
ci:test:sf

File: test.properties [hunk 1]
```diff
@@ -5996,14 +5996,7 @@
...
     test.batch.names[publications]=\
-        central-requirements-jdk8,\
-        functional-tomcat90-mysql57-jdk8,\
-        functional-upgrade-tomcat90-mysql57-jdk8,\
-        js-unit-jdk8,\
-        modules-integration-mysql57-jdk8,\
-        modules-semantic-versioning-jdk8,\
-        modules-unit-jdk8,\
-        semantic-versioning-jdk8
+        functional-tomcat90-mysql57-jdk8
```
Comment (gaborlovasdotcom) [inline test.properties:5999]: @weller0246 Could you please remove this temp change also? :)

**Comment by gaborlovasdotcom** [comment]
ci:stop

**Comment by gaborlovasdotcom** [comment]
ci:test:publicaitons:nocompile

**Comment by gaborlovasdotcom** [comment]
ci:test:publications:nocompile

**Comment by gaborlovasdotcom** [comment]
Discarded as Publications team working on a generic solution in [LRQA-75717](https://issues.liferay.com/browse/LRQA-75717)

**Pull request closed without merging**